app/models/package.rb

```diff
@@ -48,6 +48,7 @@ class Package < ActiveRecord::Base
   scope :received, -> { where(state: 'received') }
   scope :expecting, -> { where(state: 'expecting') }
   scope :inventorized, -> { where.not(inventory_number: nil) }
+  scope :hasZeroQuantity, -> { where(quantity: 0) }
```

Comment:
Also can you rename the scope to qtyPackage or something? Because {{hasZeroQuantity}} sounds like we're returning boolean value, which is not true